distribution/docs/src/main/resources/_contents/_security-framework/developing-token-validators-contents.adoc

```diff
@@ -0,0 +1,9 @@
+
+==== Developing Token Validators
```

Comment:
PolicyExtension, yes, but I don't think I want anyone else making a SecurityAssertion impl. We probably should have made that internal.